### PR TITLE
SAR Content Changes

### PR DESCRIPTION
**Pull request by ajaitasaini**
### Description

as title!

### Related ticket(s)

CMDCT-

---
### How to test

Follow the [Figma](https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=2186-39323&mode=design&t=tGbsectUs8UXCn0X-4) to confirm all content changes in SAR + SAR admin have been addressed

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/verbiage/pages/accordion.ts [hunk 1]
```diff
@@ -112,47 +112,34 @@ export default {
...
-            "<br>This reporting tool is to be used by grantees for reporting of MFP program data. The information provided in this report will allow CMS to monitor grantee progress and identify challenges and improvement opportunities. For additional guidance on completing this form, please see the associated ",
+            "<br><br>Click “Add new MFP SAR submission” below to begin reporting for the current reporting period. For additional guidance on completing this form, see the associated ",
         },
         {
           type: "externalLink",
-          content: "User Guide",
+          content: "User Guide and Help File",
           props: {
-            href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(k)", // TO-DO: currently placeholder, replace when correct link available
+            href: "https://www.google.com",
```
Comment (gmrabian) [inline services/ui-src/src/verbiage/pages/accordion.ts:128]: switching back to a placeholder?

File: services/app-api/forms/sar.json [hunk 2]
```diff
@@ -1151,13 +1301,14 @@
             "isRequired": true,
             "verbiage": {
               "intro": {
-                "section": "Expenditures",
+                "section": "State- or Territory-Specific Initiatives",
```
Comment (gmrabian) [inline services/app-api/forms/sar.json:1304]: We going back to `State-` here or should it just be `State` (without hyphen)?

File: services/app-api/forms/sar.json [hunk 3]
```diff
@@ -926,22 +1088,14 @@
...
-        "tableHeader": "Initiative name <br/> Work Plan topic",
+        "dashboardSubtitle": "Your initiatives are auto-populated from your most recent approved MFP Work Plan. Note: Initiative topics with * are required initiatives. <br><br>Recipients must report on the progress of initiatives that were ongoing during the current reporting period. For each initiative, enter information on expenditures and activities, whether continuing from prior reporting periods or initiated during this reporting period. <br><br> For each initiative, recipients must also report on progress toward achieving the objective(s) identified in the MFP Work Plan. Progress toward these objectives indicates the state or territory’s greater ability to provide home and community-based services (HCBS) instead of services in institutional settings. <br><br>If your state or territory has not achieved the targets for performance measures or expected time frames for deliverables set in the MFP Work Plan, use the following questions to explain the barriers or challenges that have hindered progress and describe plans to address them.",
```
Comment (gmrabian) [inline services/app-api/forms/sar.json:1097]: 1. do we need an aria label on the asterisk? I know we've done that elsewhere
2. Remove spaces before and after the line breaks (specifically the breaks before "For each initiative..." and after "...services in institutional settings."

**Review by gmrabian — COMMENTED** [review]
Looking good! Some nits/questions and then ready to approve

File: services/ui-src/src/verbiage/pages/accordion.ts (hunk 1 above)
Comment (ajaitasaini) [reply services/ui-src/src/verbiage/pages/accordion.ts:128]: I don't actively remember making this change tbh, but the old link was also a placeholder so either works!

File: services/app-api/forms/sar.json (hunk 2 above)
Comment (ajaitasaini) [reply services/app-api/forms/sar.json]: thanks for the catch!

**Review by gmrabian — APPROVED** [review]

File: services/ui-src/src/utils/testing/mockFullReportJSON.tsx [hunk 4]
```diff
@@ -698,7 +698,7 @@ export const mockFullReportJSON: ReportJson = {
             readOnlyEntityDetailsButtonText: "View",
             dashboardTitle: "Initative total count",
             countEntitiesInTitle: true,
-            tableHeader: "Initaitve name <br/> Work Plan topic",
+            tableHeader: "Initaitve name <br/> MFP Work Plan topic",
```
Comment (jessabean) [inline services/ui-src/src/utils/testing/mockFullReportJSON.tsx:701]: Looks like there is a typo here... The beginning of this should read `Initiative`

**Review by gmrabian — APPROVED** [review]

**Review by jessabean — APPROVED** [review]

**Pull request merged**